<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QWidget</class>
 <widget class="QWidget" name="QWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QFrame" name="fr4">
     <property name="styleSheet">
      <string notr="true">#lbl3,#lbl4,#lbl2{
color: rgb(255,255,255)
}
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="fr5">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="lbl2">
           <property name="text">
            <string>From Date</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="FromDate"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="fr6">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="lbl4">
           <property name="text">
            <string>To Date</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="ToDate"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="fr7">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="lbl3">
           <property name="text">
            <string>Hour</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Hour">
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>100</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_9">
        <property name="styleSheet">
         <string notr="true">#NonSelectedButton,#SelectedButton,#PendingButton{
background-color: rgb(123,97,255)

}
#NonSelectedButton,#SelectedButton,#PendingButton{
color: rgb(255,255,255)

}

</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <widget class="QCheckBox" name="SelectedButton">
           <property name="text">
            <string>Selected Deals</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="NonSelectedButton">
           <property name="text">
            <string>Non-Selected Deals</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="PendingButton">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Pending Deals</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="fr10">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QTableWidget" name="table">
           <property name="styleSheet">
            <string notr="true">#table{
color: rgb(0,0,0)
}</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbl5">
        <property name="text">
         <string>* if u want for all the hours , leave the hours column blank</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
